CONTRIBUTING.md

```diff
@@ -1,61 +1,72 @@
+This makes them ideal for newer developers, or those who are new to these libraries and want to see how the contribution process works.
 
-You can also work on any other issue you choose to.
-The "good first" and "good to take" issues are just issues where we have a clear picture about scope and timeline.
-Pull requests working on other issues or completely new problems may take a bit longer to review when they don't fit into our current development cycle.
+We also have a list of [good to take issues](https://github.com/mui/material-ui/issues?q=is:open+is:issue+label:"good+to+take"), which are issues that have already been at least partially resolved in discussion, to the point that it's clear what to do next.
+These issues are great for developers who want to reduce their chances of falling down a rabbit hole in search of a solution.
 
-If you decide to fix an issue, please make sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you have started to work on it, so other people don't accidentally duplicate your effort.
+Of course, you can work on any other issue you like—the "good first" and "good to take" issues are simply those where the scope and timeline may be better defined.
+Pull requests for other issues, or completely novel problems, may take a bit longer to review if they don't fit into our current development cycle.
 
-If somebody claims an issue but doesn't follow up for more than a week, it's fine to take it over, but you should still leave a comment.
-If there has been no activity on the issue for 7 to 14 days, it is safe to assume that nobody is working on it.
+If you decide to fix an issue, please make sure to check the comment thread in case somebody is already working on a fix.
+If nobody is working on it at the moment, please leave a comment stating that you've started to work on it, so other people don't accidentally duplicate your effort.
 
-## Sending a Pull Request
+If somebody claims an issue but doesn't follow up after more than a week, it's fine to take over, but you should still leave a comment.
+If there has been no activity on the issue for 7 to 14 days, then it's safe to assume that nobody is working on it.
 
-MUI is a community project, so Pull Requests are always welcome, but, before working on a large change, it is best to open an issue first to discuss it with the maintainers.
+## Sending a pull request
 
-When in doubt, keep your Pull Requests small. To give a Pull Request the best chance of getting accepted, don't bundle more than one feature or bug fix per Pull Request. It's often best to create two smaller Pull Requests than one big one.
+MUI is a community project, so pull requests are always welcome, but before working on a large change, it's best to open an issue first to discuss it with the maintainers.
```

Comment:
Referring to MUI as a project sounds the same to me as "Material UI is a community project". Maybe we can expand the scope here to mention all the Core family of projects/products (and which one, out of these two, should we use, by the way? 🤔)

```suggestion
MUI Core projects are community-led, so pull requests are always welcome, but before working on a large change, it's best to open an issue first to discuss it with the maintainers.
```

Could also be, maybe:

```suggestion
MUI Core projects are community-driven, so pull requests are always welcome, but before working on a large change, it's best to open an issue first to discuss it with the maintainers.
```